lib/app_api/captures/capture.ex

```diff
@@ -17,5 +24,24 @@ defmodule AppApi.Captures.Capture do
     capture
     |> cast(attrs, [:id_person, :text, :completed])
     |> validate_required([:id_person, :text, :completed])
+    |> put_assoc(:tags, parse_tags(attrs))
+  end
+
+  defp parse_tags(params) do
+    (params["tags"] || "")
+    |> String.split(",")
+    |> Enum.map(&String.trim/1)
+    |> Enum.reject(&(&1 == ""))
+    |> insert_and_get_all()
+  end
+
+  defp insert_and_get_all([]) do
+    []
+  end
+
+  defp insert_and_get_all(names) do
+    maps = Enum.map(names, &%{text: &1})
+    Repo.insert_all(Tag, maps, on_conflict: :nothing)
```

Comment:
A conflict can appears when inserting tags, for example on a race condition. If mutiple tag are inserted at the same time with the same name the postgres index will report an error. The `on_conflict` catch this error and do nothing as at least one tag will be inserted with the correct name.

Reply:
@SimonLab ideally the index should be based on the person_id and tag_id so that different people can have the _same_ tag. But I think we need a dedicated issue for tag ownership. 💭